WiCAN_GUI/GUI/GUI_methods.py

```diff
@@ -263,13 +266,14 @@ def __init__(self):
         # Connect Buttons
         self.play_pause_btn.clicked.connect(self.toggle_pause)
         self.dbc_filediag_btn.clicked.connect(self.get_dbc_path)
-        self.send_msg_btn.clicked.connect(self.send_data_table.send_msg)
+        if supports_tx:
+            self.send_msg_btn.clicked.connect(self.send_data_table.send_msg)
 
     def update_plot(self):
         self.data_plotter.update_plot(self.signal_table.get_selected())
 
     def update_time(self):
-        self.CurrTimeLabel.setText(f"Current Time: {time.clock():.2f}s")
+        self.CurrTimeLabel.setText(f"Current Time: {time.perf_counter():.2f}s")
```

Comment:
I doubt we want this, but I added this workaround. The issue is `time.clock()` is deprecated on latest python